packages/generator/src/sdk-metadata/sdk-metadata.spec.ts

```diff
@@ -29,7 +29,7 @@ describe('sdk-metadata', () => {
       createOptions({ versionInPackageJson: '1.0.0' })
     );
     expect(metaData).toMatchSnapshot();
-    expect(metaData.serviceStatus).toBe('certified');
+    expect(metaData.serviceStatus.status).toBe('certified');
```

Comment:
Maybe you can add a test for the `statusText`.

Reply:
done